ReactNativeClient/lib/components/screens/config.js

```diff
@@ -101,27 +103,29 @@ class ConfigScreenComponent extends BaseScreenComponent {
 			const exportPath = this.state.profileExportPath;
 			const resourcePath = `${exportPath}/resources`;
 			try {
+				const hasPermissions = await checkPermissions(PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE);
+				if (!hasPermissions) {
+					throw new Error('Permission denied');
+				}
```

Comment:
Does this work on iOS? (since it's referring to an Android-specific package)

Reply:
Good catch, thanks. I assumed there should be a platform check.
The check is there alright, it's just that for iOS the method will always return false:
https://github.com/facebook/react-native/blob/c0d8c1db9077d08c3832b23e959bf70ad204307d/Libraries/PermissionsAndroid/PermissionsAndroid.js#L140

Fixed now.